### PR TITLE
shogimaru 1.3.1 (new formula)

### PR DESCRIPTION
**Pull request by shogimaru**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/shogimaru.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  def install
+    system "qmake", "-recursive", "CONFIG+=release", "target.path=#{bin}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    if OS.mac?
+      otool "-L", "#{bin}/shogimaru"
```
Comment (SMillerDev) [inline Formula/shogimaru.rb:25]: I don't think this will really validate that it works correctly though.

File: Formula/shogimaru.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  depends_on "qt"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
+  def install
+    system "qmake", "-recursive", "CONFIG+=release", "target.path=#{bin}"
```
Comment (SMillerDev) [inline Formula/shogimaru.rb:18]: Is it not ported to cmake yet?

Comment (shogimaru) [reply Formula/shogimaru.rb:18]: Currently, only 'qmake' is available.

File: Formula/shogimaru.rb (hunk 1 above)
Comment (shogimaru) [reply Formula/shogimaru.rb:25]: I have made first Formula pull-request on Linux, I don't have a mac.

Could you tell me how to validate a GUI application on brew?
Should it be checked if it is possible to start the application?

Comment (SMillerDev) [reply Formula/shogimaru.rb:25]: That would be a better test, yes.

**Comment by shogimaru** [comment]
Errors could not be resolved.

**Pull request closed without merging**